TeachingRecordSystem/src/TeachingRecordSystem.Worker/appsettings.Development.json

```diff
@@ -1,16 +1,17 @@
 {
-  "DqtReporting": {
-    "ProcessAllEntityTypesConcurrently": false
-  },
-  "Gias": {
-    "RefreshEstablishmentsJobSchedule": "0 0 31 2 *"
-  },
-  "StorageConnectionString": "UseDevelopmentStorage=true",
-  "TrsSyncService": {
-    "ModelTypes": [
-      "Person",
-      "MandatoryQualification"
-    ],
-    "IgnoreInvalidData": true
-  }
+    "DqtReporting": {
```

Comment:
Nit: can we undo the indenting changes here? (I know editorconfig is currently borked which is probably the source)